apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -1231,9 +1234,9 @@ function initPort( message ) {
 	window.removeEventListener( 'message', initPort, false );
 }
 
-$( () => {
+( function () {
```

Comment:
Good question! One thought: it's beneficial for these functions to execute as soon as possible so that the parent frame knows that the iframe has loaded. I think that this approach would execute more quickly than dom ready. Also, the functionality in this file can't rely on DOM content staying the same, since everything is managed in React. So I don't think the script needs to wait for execution until that event.